package.json

```diff
@@ -180,6 +179,7 @@
     "clean-git-ref": "2.0.1",
     "commander": "10.0.1",
     "conventional-commits-detector": "1.0.3",
+    "cron-parser": "4.8.1",
```

Comment:
We have an issue where @rarkins says to use the `cronstrue` npm package:

- #22267

I don't know if `cron-parser` can do something similar.

I'll let you maintainers decide which package to use. 😄

Reply:
It has some limitations so I picked cron-parser😶. But sure we can use it for printing if that's okay?

Reply:
I think we can add the visual part later, this is fine for now